slides/aml-06-020518-linear-models-classification/index.html

```diff
@@ -0,0 +1,689 @@
+        font-family: 'Garamond';
+        font-weight: normal;
+        margin-top: 10px;
+        margin-bottom: 10px;
+      }
+      .remark-slide-content h1 {
+        font-size: 70px;
+        text-align: center;
+      }
+      .remark-slide-content p, .remark-slide-content li {
+        font-size:30px;
+        line-height: 1.4;
+      }
+      .remark-code {
+        font-size:30px;
+      }
+      .remark-slide-content p {
+          margin: 5px;
+      }
+      .remark-slide-container .spacious p,
+      .remark-slide-container .spacious li{
+          margin-bottom: 50px;
+          margin-top: 50px;
+      }
+      .remark-slide-container .spacious h1{
+          margin-bottom: 50px;
+      }
+      .remark-slide-container .some-space p,
+      .remark-slide-container .some-space li,
+      .remark-slide-container .some-space h1{
+          margin-bottom: 30px;
+      }
+      .reset-column {
+          overflow: auto;
+          width: 100%;
+      }
+      .remark-slide-content .compact p, .remark-slide-content .compact li, .remark-slide-content .compact pre{
+          font-size: 30px;
+          line-height: 1.1;
+          display: block;
+          margin: 2px 0;
+      }
+      .padding-top {
+          padding-top: 100px;
+      }
+      .remark-slide-content .smaller p, .remark-slide-content .smaller li,
+      .remark-slide-content .smaller .remark-code, .remark-slide-content .smaller a{
+          font-size: 25px;
+      }
+      .normal {
+          font-size: 30px;
+      }
+      .quote_author {
+          display: block;
+          text-align: right;
+          margin-top: 20px;
+          font-size: 30px;
+          font-family: 'Garamond';
+      }
+      .larger, .larger .remark-code {
+          font-size: 40px;
+      }
+      .largest, .largest .remark-code {
+          font-size: 50px;
+      }
+      .left-column, .right-column {
+          width: 48%;
+      }
+      .right-column{
+          float: right;
+      }
+      .left-column{
+          float: left;
+      }
+      .narrow-right-column {
+          float: right;
+          width: 32%
+      }
+      .wide-left-column {
+          float: left;
+          width: 65%
+      }
+      .invisible {
+          visibility: hidden
+      }
+      .remark-code, .remark-inline-code { font-family: 'Ubuntu Mono'; }
+    </style>
+  </head>
+  <body>
+    <textarea id="source">
+
+class: center, middle
+
+### W4995 Applied Machine Learning
+
+# Linear Models for Classification
+
+02/06/17
+
+Andreas C. Müller
+
+???
+N/A
+
+---
+class: center, middle
+
+Linear models for <strong>binary</strong> classfication
+
+???
+N/A
+
+---
+class: spacious
+
+.center[
+![:scale 60%](images/img_1.png)
+]
+
+.smaller[
+$$\hat{y} = sign(w^T \textbf{x} + b) = sign(\sum\limits_{i}w_ix_i + b)$$
+]
+
+???
+
+---
+
+class: left, middle
+
+# Picking a loss?
+
+.medium[
+$\hat{y} = sign(w^T \textbf{x} + b)$
+
+`$\min_{w \in ℝ^{p}}$$\sum_{i=1}^n$`$1_{y_i \neq sign(w^T \textbf{x} + b)}$
```

Comment:
Did you write this by hand? I find it a bit odd to mix unicode and latex and break up the line into two formulas (and the first half is escaped?)

Reply:
I'd prefer display mode (`$$`) for large equations.

Reply:
Yes I did. So remark uses mathjax to render latex - math and it has a subscript issue. Here is the link
https://github.com/mathjax/MathJax/issues/329

It does not recognize subscript (_) if used to put more than one variable in subscript. So I tried a few things and added that. The gap between 2 equations was to make sure they were separate without failing to rander

Reply:
```
`$$\min_{w \in ℝ^{p}} \sum_{i=1}^n 1_{y_i \neq sign(w^T \textbf{x} + b)}$$`
```
works for me

Reply:
I suggest you look at how I did it in previous slides if you're unsure about something ;)

Reply:
I'll push a commit with some minor formatting in a bit, you can see what I'd do differently. Either way, thanks for your help!

Reply:
Cool ! No problem :) 

Regarding your previous comment on adding code from the notebook, I am assuming you mean it for parts where there are code snippets. I would still have to use images for code-generated tables or actual images. Please correct me if I am wrong.

Reply:
Indeed. At one point I tried to make the table html but I don't think it's worth it.
For images I started out just storing the images from the presentation, but then moved to saving the images from the notebook using savefig with 300 dpi, which gives nicer images.

Btw, do you have the formula plugin for libreoffice? That should allow you to copy and paste the latex for the formulas.

Reply:
Oh no, I don't have it and wasn't aware the equations were written in that format. I thought those were just images. I will install it and that should help me a lot. Thanks!